tests/integrationtest/t/planner/core/casetest/predicate_simplification.test

```diff
@@ -31,3 +31,112 @@ explain format = 'brief' select count(*) cnt from ts where f <> '1' and f in ('1
 explain format = 'brief' select f from t where f <> NULL  and f in (1,2,3) -- Special case of NULL with no simplification.;
 explain format = 'brief' select f from t where f != NULL  and f in (NULL,2,3) -- Special case of NULL with no simplification.;
 
+# Test OR simplification start
+# Airbnb test case
```

Comment:
It is best not to include the customer's name

Reply:
Agree and removed.